docs/application/vscode-ext/Tizen/hybrid.md

```diff
@@ -0,0 +1,223 @@
+
+Hybrid app successfully created with selected working folder as main app and selected  dependency projects to main app as sub apps.
+
+
+### Build your project
+
+1. Open the **Command Palette** and enter build. Select **Tizen: Build Tizen project**.
+
+   ![Build project](media/hybrid_project_build.png)
+
+2. Review the build results in the output window, and check the location of the package file (.tpk).
+
+   ![Review result](media/hybrid_build_result.png)
+
+### Deploy and run your application in emulator
+
+1. To launch the Tizen Emulator Manager, open the **Command Palette** and enter emulator. Select **Tizen: Launch Tizen Emulator Manager**.
+
+   ![Launch emulator](media/hybrid_deploy.png)
+
+2. Create and launch an emulator instance in the Emulator Manager.
+
+3. To deploy your application to the target, enter install Tizen in the **Command Palette** and select **Tizen: Install Tizen application**.
+
+   ![Install application](media/hybrid_install_application.png)
+
+   ![Install application](media/hybrid_install_result.png)
+
+4. To run the application on the emulator, enter run in the **Command Palette** and select **Tizen: Run Tizen application**.
+
+   ![Run application](media/hybrid_run_application.png)
+
+   ![application](media/hybrid_run_result.png)
+
+### Debug your application in emulator
+
+1. Deploy and run your application in the emulator.
+
+2. In the Visual Studio Code Activity Bar, select the **Debug** icon.
+
+3. In the Debug view, open the Configuration drop-down menu and select **Add Configuration** or select **Create a Launch.json file**.
+
+4. Select **GDB for Tizen Native** in the list of configurations.
+
+   ![Debugger](media/hybrid_debug.png)
+
+5. Add a break point in your source code.
+
+   ![Break point](media/hybrid_add_breakpoint.png)
+
+6. Start the debugging session by pressing **F5**.
+
+   ![Start debug](media/hybrid_select_start_debug.png)
+
+   ![Start debug](media/hybrid_start_debug.png)
+
+### Remove Dependency projects
+
+- Already added Dependency projects(sub apps) can be removed  from main app
+
+1. From vscode File explorer Right click on **main app** and select **Tizen:Update Dependency Projects**.
+
+   ![Selected dependencies projects](media/selected_dependency_workspace_yaml.png)
+
+   ![remove dependency project](media/remove_update_dependency_project.png)
+
+2. Quick pick list with project list for select/deselect  will be displayed.
+
+   ![Display depencency project list](media/deselect_dependency_project.png)
+
+3. DeSelect the dependency projects and press OK button.
+
+   ![Deselect dependency project](media/deselected_dependency_project.png)
+
+4. removed dependencies will be updated in workspace.yaml file
+
+   ![Removed dependencies projects](media/removed_dependency_workspace_yaml.png)
+
+Deselected projects will be removed from dependency project list from main app.
+
+## Import Tizen project
+
+### Import Tizen workspace Folder

```

Comment:
The word “Folder” shouldn’t be capitalized to maintain consistency with the rest of the document.
```suggestion
### Import Tizen workspace folder

```